resources/public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,385 @@
+openapi: 3.0.3
+info:
+  title: CTC Traders Test Support
+  contact: {}
+  version: '1.0'
+  description: |
+    # Overview
+    This API allows testers to inject departure and arrival movement notifications as 
+    if they have come from the office of departure or the office of destination by the New Computerised Transit System (NCTS).

```

Comment:
Need to bring the start of the overview into line with the OpenAPI definitions for CTC Traders API and CTC Guarantee Balance API. The wording  used for intro in [v2 landing page](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/common-transit-convention-traders-test-support/2.0) should be reused here.

Suggestion:
Use the CTC Traders Test Support API to  inject departure and arrival movement notifications as if they have come from a customs office of departure or destination through the New Computerised Transit System (NCTS).